docs/preview/features/logging.md

```diff
@@ -61,6 +61,57 @@ app.UseEndpoints(endpoints => endpoints.MapControllers());
 If you have multiple middleware components configured, it is advised to put the `ExceptionHandlingMiddleware` as soon as possible.
 By doing so, unhandled exceptions that might occur in other middleware components will also be logged by the `ExceptionHandlingMiddleware` component.
 
+### Configuration
+
+When custom exception handling is required, you can inherit from the `ExceptionHandlingMiddleware` to create your own middleware component and register it with Arcus' extension.
+
+This example implements the exception handling middleware to influence the log message and returns a custom determination of the HTTP response status code.
+
+```csharp
+using Arcus.WebApi.Logging;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+
+public class MyExceptionHandlingMiddleware : ExceptionHandlingMiddleware
+{
+    public MyExceptionHandlingMiddleware(RequestDelegate next) : base(next)
+    {
+    }
+
+    protected override void LogException(ILoggerFactory loggerFactory, Exception exception)
+    {
+        ILogger logger = loggerFactory.CreateLogger<MyExceptionHandlingMiddleware>();
+        logger.LogCritical(exception, "Custom exception handling!");
+    }
+
+    protected override HttpStatusCode DetermineResponseStatusCode(Exception exception)
+    {
+        if (exception is ValidationException)
+        {
+            return HttpStatusCode.BadRequest;
+        }
+
+        if (exception is TimeoutException)
+        {
+            return HttpStatusCode.ServerTimeout;
+        }
+
+        return HttpStatusCode.InternalServerError;
+    }
```

Comment:
This sample will need to be adapted to reflect the new situation.

Reply:
Ah, yes (stupid me 😅 ).

Reply:
Done in https://github.com/arcus-azure/arcus.webapi/pull/342/commits/989708ae0bfd9dcdfb95e0304f0ec51e082144aa

Reply:
This looks good!